python/python/glide/async_commands/core.py

```diff
@@ -694,6 +694,29 @@ async def lpush(self, key: str, elements: List[str]) -> int:
             int, await self._execute_command(RequestType.LPush, [key] + elements)
         )
 
+    async def lpushx(self, key: str, elements: List[str]) -> int:
+        """
+        Inserts specified values at the head of the list, only if `key` already exists and holds a list.
+
+        See https://redis.io/commands/lpushx/ for more details.
+
+        Args:
+            key (str): The key of the list.
+            elements (List[str]): The elements to insert at the head of the list stored at `key`.
+
+        Returns:
+            int: The length of the list after the push operations.
+
+        Examples:
+            >>> await client.lpushx("my_list", ["value1", "value2"])
+                2
```

Comment:
command works with existing lists only and returns the resulting length
maybe worth adding a comment
```suggestion
                3
```

update for rpushx too

Reply:
The doc string "Inserts..." is a PARTIAL copy-paste from the redis.io doc. Please add the missing sentence - no reason to have only a partial doc :)

Also the return value in the example indeed should be 3

Reply:
Some docs are rather big and we are copying only the most important part.
See java implementation for these commands in #1225